internal/pkg/policy/policy_output.go

```diff
@@ -238,10 +242,7 @@ func (p *Output) prepareElasticsearch(
 	// in place to reduce number of agent policy allocation when sending the updated
 	// agent policy to multiple agents.
 	// See: https://github.com/elastic/fleet-server/issues/1301
-	if err := setMapObj(outputMap, output.APIKey, p.Name, "api_key"); err != nil {
-		return err
-	}
-
+	outputMap[p.Name]["api_key"] = output.APIKey
```

Comment:
@michel-laterman it seems to me it is going to update the parsedPolicy output here and potentially impact other checkin no?